include/ebpf_api.h

```diff
@@ -354,6 +354,17 @@ extern "C"
     uint32_t
     ebpf_api_link_program(ebpf_handle_t program_handle, ebpf_attach_type_t attach_type, ebpf_handle_t* link_handle);
 
+    /**
+     * @brief Detach the eBPF program from the link.
+     *
+     * @param[in] link_handle Handle to the link.
+     *
+     * @retval ERROR_SUCCESS The operations succeeded.
+     * @retval ERROR_INVALID_PARAMETER One or more parameters are incorrect.
```

Comment:
```suggestion
     * @retval ERROR_INVALID_PARAMETER The link handle is invalid.
```
There's only one parameter, so the "or more" doesn't apply.

Reply:
Fixed.